client/src/components/Vote.svelte

```diff
@@ -60,7 +61,9 @@
           on:click={() => vote(lobby.uids[i])}
         >
           <AvatarImg {avatar} />
-          <span class="mdc-typography--subtitle1">{displayName ?? ""}</span>
+          <span class="mdc-typography--subtitle1 {catfishes?.includes(lobby.uids[i]) ? 'catfish' : ''}"
```

Comment:
this should use `class:` like this
https://github.com/cat-acronym-team/cat-me-if-you-can/blob/feba43c513a5c303f605f707598be9a2786fcaf3/client/src/components/ChatMessages.svelte#L69-L71